tests/test_quantum_gradients.py

```diff
@@ -468,6 +468,44 @@ def d_error(p, grad_method):
         assert grad_fd1 == pytest.approx(grad_auto, abs=tol)
         assert grad_angle == pytest.approx(grad_auto, abs=tol)
 
+
+    def test_hybrid_gradients_numpy(self, qubit_device_2_wires, tol):
```

Comment:
Should we add `autograd` to the test name just to make it clear?

Reply:
Sure.